Change descriptions (all inspired by games-misc/opengfx-0.4.7):

1. Add the -P flag for GCC 4.8 compatibility
2. Add the bundle make target again
3. Disable PNG regeneration with GIMP because it "causes sandbox violations and not worth the effort anyway"

--- Makefile
+++ Makefile
@@ -184,7 +184,7 @@
 
 %.nml: %.pnml
 	$(_E) "[CPP] $@"
-	$(_V) $(CC) -D REPO_REVISION=$(NEWGRF_VERSION) -D NEWGRF_VERSION=$(NEWGRF_VERSION) $(CC_USER_FLAGS) $(CC_FLAGS) -o $@ $<
+	$(_V) $(CC) -P -D REPO_REVISION=$(NEWGRF_VERSION) -D NEWGRF_VERSION=$(NEWGRF_VERSION) $(CC_USER_FLAGS) $(CC_FLAGS) -o $@ $<
 
 clean::
 	$(_E) "[CLEAN NML]"
@@ -395,6 +395,8 @@
 	$(_E) "[CLEAN GRFID]"
 	$(_V) -rm -f $(MD5_FILENAME)
 
+bundle: $(DIR_NAME)
+
 # Bundle directory
 $(DIR_NAME): $(BUNDLE_FILES)
 	$(_E) "[BUNDLE] $@"
--- Makefile.config
+++ Makefile.config
@@ -36,7 +36,7 @@
 
 # Uncomment in order to make use of gimp scripting. See the file
 # for a description of the format
-GFX_SCRIPT_LIST_FILES      := sprites/png/terrain/waterfeatures/waterfeatures.xcf2png sprites/png/infrastructure/locks.xcf2png sprites/png/terrain/terrain.xcf2png sprites/png/trees/trees.xcf2png sprites/png/industries/coalmine.xcf2png sprites/png/industries/goldmine/goldmine.xcf2png sprites/png/industries/oilwell/oilwell.xcf2png
+#GFX_SCRIPT_LIST_FILES      := sprites/png/terrain/waterfeatures/waterfeatures.xcf2png sprites/png/infrastructure/locks.xcf2png sprites/png/terrain/terrain.xcf2png sprites/png/trees/trees.xcf2png sprites/png/industries/coalmine.xcf2png sprites/png/industries/goldmine/goldmine.xcf2png sprites/png/industries/oilwell/oilwell.xcf2png
 
 # If needed, declare the minimum NML requirements
 # REQUIRED_NML_BRANCH  = 0.3
